dvc/dvcfile.py

```diff
@@ -39,11 +37,6 @@ def __init__(self, path, pipeline_file=False):
         )
 
 
-class LockfileCorruptedError(DvcException):
```

Comment:
Got rid of LockfileCorruptedError and StageFileFormatError.

Reply:
To clarify, this was removed because the pretty_print is implemented on the `YAMLValidationError` side, it's quite tricky to pass that context of data/text/path, etc. to other exceptions that are reraised. And, that too in two of these exceptions.